src/ontology/mondo-edit.obo

```diff
@@ -584028,6 +584030,15 @@ is_a: MONDO:0007015 {source="PMID:36304598"} ! viral meningitis
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-3458-4839
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7737" xsd:anyURI
 
+[Term]
+id: MONDO:1030005
+name: mosaic SMO syndrome
+def: "A somatic mosaic condition caused by a postzygotic mutation (c.1234 C>T p.Leu412Phe) in the SMO gene. Two overlapping clinical syndromes associated with this variant in SMO have been reported; Curry-Jones Syndrome (MIM:601707) and Happle-Tinschert Syndrome.\n\nThe syndrome is characterized by cutaneous and skeletal manifestations such as linear hypo‐ or hyperpigmented lesions, basaloid follicular hamartomas, palmoplantar pitting, atrophoderma, hypertrichosis, polydactyly or syndactyly, rib anomalies (rudimentary ribs), and limb‐length anomalies. Craniofacial and dental abnormalities such as dysmorphic facies, macrocephaly, craniosynostosis, jaw tumors (ameloblastoma) have also been reported. \n\nAdditional abnormalities impacting the gastrointestinal, cerebral, ophthalmic, and gonadal organs should also be considered. Some of these features include anal anomaly, colonic adenocarcinoma, severe constipation, myelofibrosis and smooth muscle hamartomas, malrotation, medulloblastoma, cerebral malformations, developmental delay, cataract, microphthalmia, coloboma, glaucoma, and cryptorchidism. \n\nLumping these disorders into one disease entity is justified by the common molecular mechanism and variant and phenotypic overlap. Based on the mosaic nature of this condition, different clinical presentations are expected and the two diseases should be considered as a spectrum of one disorder." [https://clinicalgenome.org/affiliation/40059/, PMID:27236920, PMID:30497210, PMID:32413283, PMID:7606318]
+is_a: MONDO:0015160 {source="https://orcid.org/0000-0002-3458-4839"} ! multiple congenital anomalies/dysmorphic syndrome-variable intellectual disability syndrome
```

Comment:
After reading more articles, I think the parent of MONDO:0019042 'multiple congenital anomalies/dysmorphic syndrome' is better since the variable intellectual aspect is not highlighted in all references as compared to the multiple congenital anomalies. I think this is the better reference: PMID: 31120550 (which is not one of the original PMIDs in the issue).